packages/bundle-source/CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [3.2.0](https://github.com/endojs/endo/compare/@endo/bundle-source@3.1.0...@endo/bundle-source@3.2.0) (2024-03-19)
+
+
+### Features
+
+* **ses-ava:** import test from @endo/ses-ava/prepare-endo.js ([#2133](https://github.com/endojs/endo/issues/2133)) ([9d3a7ce](https://github.com/endojs/endo/commit/9d3a7ce150b6fd6fe7c8c4cc43da411e981731ac))
```

Comment:
not a consumer-facing feature

Reply:
I think this is a user-facing feature, in that the user needs to do a minor refactor to take advantage of this new public export. Once they have done this refactor, no prior version will suffice.

Reply:
It's user-facing for ses-ava but not for bundle-source. There are no code changes in bundle-source. It should have a message like base64 has:
> **Note:** Version bump only for package @endo/base64

But maybe this comment is superseded by your PR comment.